player-avatar-binding.js

```diff
@@ -157,6 +164,30 @@ export function applyPlayerActionsToAvatar(player, rig) {
   // pose
   rig.poseAnimation = poseAction?.animation || null;
 }
+export function applyPlayerMirrorsToAvatar(player, rig, mirrors) {

```

Comment:
re mirrors sort: In the future would it be worth adding a collider to certain interactive objects and on collision we can say its the active Mirror or closest mirror?

Reply:
We could indeed do it by distance to a collider.

But it might be more robust for the API to talk in terms of Object3D. Any user of this API can convert physics to an Object3D. Asking a client to generate physics just to get a mirror sounds like a worse experience.